Fix id type differences in browser and code

Tommi377 opened the pull request:
Based on #44 

more info:
https://trello.com/c/VHll3BH9/150-refactor-the-ids-to-be-uniform-in-code-and-data

10hoe reviewed (approved):
The last commit LGTM, and the projects seem to work well.

jaakkoha-bytecraft commented:
UUIDs as primary keys is in my opinion an antipattern, unless you’re working with a distributed system with partition tolerance.

The biggest problem in our use case compared to numeric ids is that they’re hard to remember / recognize (issue-1224 vs. issue-INSERT-UUID-HERE), but there are other more minor issues as well. 

https://bytebase.com/blog/choose-primary-key-uuid-or-auto-increment

> On 8. Feb 2022, at 21.37, 10hoe ***@***.***> wrote:
> 
> ﻿
> @10hoe approved this pull request.
> 
> The last commit LGTM, and the projects seem to work well.
> 
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android. 
> You are receiving this because you are subscribed to this thread.

Tommi377 commented:
> UUIDs as primary keys is in my opinion an antipattern, unless you’re working with a distributed system with partition tolerance. The biggest problem in our use case compared to numeric ids is that they’re hard to remember / recognize (issue-1224 vs. issue-INSERT-UUID-HERE), but there are other more minor issues as well. https://bytebase.com/blog/choose-primary-key-uuid-or-auto-increment

I changed it back to SERIAL and changed the id type to number. We now have to keep in mind that the id needs to be converted to a string when using the react flow library but otherwise it'll work the same.

WatMan77 commented on cypress/integration/sample_spec.js:
```diff
@@ -59,7 +59,7 @@ describe('Login / Register', () => {
             cy.get('#login-button').click()
             
-            cy.contains('#loing-error').should('not.exist')
+            cy.contains('#loging-error').should('not.exist')
```
#login-error not #loging-error

Tommi377 replied:
This should be commented in PR #44 since the change was done there (fixed now). It might be a good idea to delay the review until PR #44 is added because the diffs are not accurate

Tommi377 commented:
Made this a stacked PR so now the diffs are correct this time
